components/screens/LoginScreen.js

```diff
@@ -0,0 +1,237 @@
+import React from 'react';
+import {
+    View,
+    StyleSheet,
+    ImageBackground,
+    Text,
+    Image,
+    TouchableOpacity
+} from 'react-native';
+
+import { LinearTextGradient } from 'react-native-text-gradient';
+import LinearGradient from 'react-native-linear-gradient';
+
+import { useNavigation } from '@react-navigation/native';
+
+import Icon from "react-native-vector-icons/FontAwesome";
+import CustomButton from '../reusable/CustomButton';
+import CustomInput from '../reusable/CustomInput';
+
+function LoginScreen() {
+    const navigation = useNavigation()
+
+    const handleCreateNewAccount = () => {
+        navigation.navigate("RegisterScreen");
+    }
+
+    const handleLogIn = () => {
+        navigation.navigate("Dashboard")
```

Comment:
it's better to set the redux state isLoggedIn to true, you should have different navigators  for when the user logged in and when he is not, so your code should look like this



```
<Stack.Navigator screenOptions={{headerShown: false}}>
      {!isLoggedIn ? (
        <Stack.Screen name="Auth" component={AuthNavigator}  />
      ) : (
        <Stack.Screen name="Main" component={MainNavigator} />
      )}
    </Stack.Navigator>
```